### ISSUE TITLE
Change Keras to tensorflow.keras

### ISSUE THREAD

**Issue by youssefoumate** [conscienceli/IterNet#5]
Hello,
I'm trying to use only tensorflow in the code, so I'm changing the Keras imports to tensorflow.keras, like this:
```
from tensorflow.keras import losses
from tensorflow.keras.callbacks import ModelCheckpoint, EarlyStopping, ReduceLROnPlateau
from tensorflow.keras.layers import Input, MaxPooling2D
from tensorflow.keras.layers import concatenate, Conv2D, Conv2DTranspose, Dropout, ReLU, BatchNormalization, Activation
from tensorflow.keras.layers import add, multiply
from tensorflow.keras.models import Model
from tensorflow.keras.optimizers import Adam
```
However, the graph gets disconnected at dropout_19_1 in the third iteration:
I get the following error:

`Graph disconnected: cannot obtain value for tensor Tensor("dropout_19_1/Identity:0", shape=(None, None, None, 32), dtype=float32) at layer "concatenate_4".`

The error is coming from this line:
`model = Model(inputs=[inputs], outputs=[outs[-1]])`

Any thoughts?

Thank you

**Comment by youssefoumate** [comment]
I had to change this line:
`conv1 = concatenate(a_layers, axis=3)` in define_model.py
to this: `conv1 = Concatenate(3)(a_layers[:])`
